crypto/src/main/kotlin/net/corda/v5/crypto/merkle/MerkleProof.kt

```diff
@@ -31,6 +31,6 @@ interface MerkleProof {
      */
     fun verify(
         root: SecureHash,
-        digestProvider: MerkleTreeHashDigestProvider
+        digestProvider: MerkleTreeHashDigest
```

Comment:
@mnesbit Can you rename it to `digest`, and generally go through the contracts and check that the names are matching after the renaming?